src/ImageDeleteModal.jsx

```diff
@@ -81,11 +75,20 @@ export const ImageDeleteModal = ({ imageWillDelete, onAddNotification }) => {
         }
     };
 
+    const imageName = repoTags[0]?.split(":")[0].split("/").at(-1) ?? "";
+
+    let isAllSelected = null;
+    if (checkedTags.length === repoTags.length)
+        isAllSelected = true;
+    else if (checkedTags.length === 0)
+        isAllSelected = false;
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1451-20231019-050221-038bc52d-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)

Reply:
This doesn't see, necessary, but I guess I can try unselecting all images for the sake of it.